component_down.py

```diff
@@ -0,0 +1,35 @@
+import os
+import time
+import json
+from collections import defaultdict
+
+agent= os.getenv('HOSTNAME')
+now = time.mktime(time.gmtime())
+since = 7 ## in days
+
+base_dir = '/data/srv/wmagent/current/install/wmagent/'
+restarts = { 
+    "timestamp" : now,
+    "since" : since,
+    "agent" : os.getenv('HOSTNAME'),
+    "data" : defaultdict(list)
+    }
+   
+for component in os.listdir(base_dir):
+    if os.path.isfile(component):continue
+    print component
+    #for grep in os.popen('grep Harness %s/%s/ComponentLog | grep Starting | grep %s'%( base_dir, component,component)):
+    for grep in os.popen('grep terminated %s/%s/ComponentLog | grep %s'%( base_dir, component,component)):
```

Comment:
please check whether this file exists first.

Reply:
What file? I ran it on both fermi and cern agent last night that part of the script works on both.